### ISSUE TITLE
ClickHouse syntax error on each query

### ISSUE THREAD

**Issue by destator** [dbeaver/dbeaver#16885]
#### System information: 
- Operating system (distribution) and version: Windows 7
- DBeaver version: 22.1.0.202206051253
- Additional extensions: None

#### Connection specification:
- Database name and version: ClickHouse 22.6.1.1985
- Driver name: com.clickhouse.jdbc.ClickHouseDriver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? : No

#### Describe the problem you're observing:
Before each user query to clickhouse dbeaver sends an extra one which fails with log message from clickhouse.
Seems like no effect to user queries.

#### Steps to reproduce, if exist:
Perform any query

#### Include any warning/errors/backtraces from the logs
ClickHouse error:
`2022.06.27 00:34:11.822167 [ 52535 ] {be708644-dd61-4f03-a15f-2658ba4b6b6e} <Error> executeQuery: Code: 62. DB::Exception: Syntax error: failed at position 72 ('Client'): Client as DEFAULT_VALUE, Application name as DESCRIPTION union all select CustomHttpHeaders as NAME, toInt32(0) as MAX_LEN,  as DEFAULT_VALUE, Custom HTTP heade. Expected one of: token, Comma, FROM, PREWHERE, WHERE, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query. (SYNTAX_ERROR) (version 22.6.1.1985 (official build)) (from 10.50.0.10:52522) (in query: select ApplicationName as NAME, toInt32(0) as MAX_LEN, ClickHouse Java Client as DEFAULT_VALUE, Application name as DESCRIPTION union all select CustomHttpHeaders as NAME, toInt32(0) as MAX_LEN, as DEFAULT_VALUE, Custom HTTP headers as DESCRIPTION union all select CustomHttpParameters as NAME, toInt32(0) as MAX_LEN, as DEFAULT_VALUE, Customer HTTP query parameters as DESCRIPTION), Stack trace (when copying this message, always include the lines below):`

**Comment by serge-rider** [comment]
Can't reproduce this. 
The error text says that you execute some long SELECT query which fails because of syntax error. This text definitely comes not from dbeaver.

How do you execute a query? 
Note that <kbd>ctrl+enter</kbd> executes currently selected query while <kbd>alt+x</kbd> executes entire script.

**Comment by destator** [comment]
I found this query - this is from clickhouse jdbc driver, method getClientInfoProperties.
Seems like some of connection parameters are undefined, which leads to the empty placeholder in query: "...toInt32(0) as MAX_LEN,  as DEFAULT_VALUE, ..."

**Comment by serge-rider** [comment]
What Clickhouse driver version do you use? You can get the version number from the Test Connection dialog.

**Comment by destator** [comment]
ClickHouse JDBC Driver clickhouse-jdbc 0.3.2 (revision: 1795272)

**Comment by emironovDB** [comment]
Can you please provide log files? You can find instruction here: https://github.com/dbeaver/dbeaver/wiki/Log-files

Also, a screenshot of the Query Manager would be handy (in case it contains the failed queries).

**Comment by destator** [comment]
```2022-06-29 16:33:10.891 - Connect with '~::ch:testdb' (com_clickhouse-181a025a773-2df0a47c513f38a2)
> Start DBeaver Generic Plug-in [org.jkiss.dbeaver.ext.generic 2.3.169.202206051253]
> Start DBeaver Clickhouse Support [org.jkiss.dbeaver.ext.clickhouse 1.0.88.202206051253]
2022-06-29 16:33:10.931 - Execution context opened (~::ch:testdb; Main; 2)
2022-06-29 16:33:11.145 - Error reading and setting client application name: Code: 62. DB::Exception: Syntax error: failed at position 72 ('Client'): Client as DEFAULT_VALUE, Application name as DESCRIPTION union all select CustomHttpHeaders as NAME, toInt32(0) as MAX_LEN,  as DEFAULT_VALUE, Custom HTTP heade. Expected one of: token, Comma, FROM, PREWHERE, WHERE, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query. (SYNTAX_ERROR) (version 22.6.1.1985 (official build))
, server ClickHouseNode(addr=http:10.50.0.1:8123, db=testdb)@-1638935699
2022-06-29 16:33:11.146 - Execution context opened (~::ch:testdb; Metadata; 3)
2022-06-29 16:33:11.153 - Error reading and setting client application name: Code: 62. DB::Exception: Syntax error: failed at position 72 ('Client'): Client as DEFAULT_VALUE, Application name as DESCRIPTION union all select CustomHttpHeaders as NAME, toInt32(0) as MAX_LEN,  as DEFAULT_VALUE, Custom HTTP heade. Expected one of: token, Comma, FROM, PREWHERE, WHERE, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query. (SYNTAX_ERROR) (version 22.6.1.1985 (official build))
, server ClickHouseNode(addr=http:10.50.0.1:8123, db=testdb)@-1638935699
2022-06-29 16:33:11.303 - Connected (com_clickhouse-181a025a773-2df0a47c513f38a2, ClickHouse JDBC Driver [clickhouse-jdbc 0.3.2 (revision: 1795272)])
2022-06-29 16:33:11.312 - Duplicate object name 'INFORMATION_SCHEMA' in cache SimpleObjectCache. Last value: information_schema
2022-06-29 16:33:11.326 - Duplicate object name 'ENUM' in cache ClickhouseDataTypeCache. Last value: Enum
2022-06-29 16:33:11.326 - Duplicate object name 'BOOL' in cache ClickhouseDataTypeCache. Last value: bool
2022-06-29 16:33:11.326 - Duplicate object name 'ARRAY(ENUM)' in cache ClickhouseDataTypeCache. Last value: `Array(Enum)`
2022-06-29 16:33:11.327 - Duplicate object name 'ARRAY(BOOL)' in cache ClickhouseDataTypeCache. Last value: `Array(bool)`
2022-06-29 16:33:11.435 - Execution context closed (~::ch:testdb, 2)
2022-06-29 16:33:11.443 - Error reading and setting client application name: Code: 62. DB::Exception: Syntax error: failed at position 72 ('Client'): Client as DEFAULT_VALUE, Application name as DESCRIPTION union all select CustomHttpHeaders as NAME, toInt32(0) as MAX_LEN,  as DEFAULT_VALUE, Custom HTTP heade. Expected one of: token, Comma, FROM, PREWHERE, WHERE, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query. (SYNTAX_ERROR) (version 22.6.1.1985 (official build))
, server ClickHouseNode(addr=http:10.50.0.1:8123, db=testdb)@-1638935699
2022-06-29 16:33:11.484 - Table 'testdb.unknown' not found in metadata catalog
2022-06-29 16:33:55.409 - Execution context closed (~::ch:testdb, 2)
2022-06-29 16:33:55.419 - Error reading and setting client application name: Code: 62. DB::Exception: Syntax error: failed at position 72 ('Client'): Client as DEFAULT_VALUE, Application name as DESCRIPTION union all select CustomHttpHeaders as NAME, toInt32(0) as MAX_LEN,  as DEFAULT_VALUE, Custom HTTP heade. Expected one of: token, Comma, FROM, PREWHERE, WHERE, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query. (SYNTAX_ERROR) (version 22.6.1.1985 (official build))
, server ClickHouseNode(addr=http:10.50.0.1:8123, db=testdb)@-1638935699
2022-06-29 16:33:55.425 - Table 'testdb.unknown' not found in metadata catalog```

**Comment by destator** [comment]
![screen](https://user-images.githubusercontent.com/7722897/176451714-3cafeaa2-39c5-4ad2-8fb0-b3992bfcdb83.png)

**Comment by zhicwu** [comment]
Apparently `ClickHouse Java Client as DEFAULT_VALUE` should be `'ClickHouse Java Client' as DEFAULT_VALUE` - same issue exists in latest patch release. I'll fix it in next release.

Update: actually it's been fixed in ClickHouse/clickhouse-jdbc#930, so upgrading driver to 0.3.2-patch9 or above will work.

**Comment by uslss** [comment]
fixed in the scope of #17879
